[BUG] The necessary packages for the template are not installed.

## Describe the bug

```sh
src/modules/src/components/user-form.test.tsx:2:23 - error TS2307: Cannot find module '@testing-library/user-event' or its corresponding type declarations.

2 import userEvent from "@testing-library/user-event";
```

When running `pnpm test` and `pnpm type` for the generated template, an error occurs.
This is because `@testing-library/user-event` package, used in generated template, is not installed.

https://github.com/monstar-lab-oss/ml-frontend/blob/58b7bffa503cd8501bcdf2b624c48e4f8fbe1790/code/react/module-api-restful/src/components/user-form.test.tsx#L2

## To Reproduce

Steps to reproduce the behavior:

1. Run `create-codes` cli to complete the template setup.
2. `pnpm i && pnpm type`
3. An error is occurring

## Expected behavior

```yaml
# in package.json
{
  # ...
  "devDependencies": {
    "@testing-library/react": "13.4.0",
    "@testing-library/user-event": "14.4.3",
    # ...
  }
}
```

## Related packages/templates

`crate-codes`

## Additional context (optional)

The options selected when exeuting the create-codes cli.

```
? Where Would You like to Create Your Application? ./my-app
? Select a JavsScript library for UI React
? Select an API Solution RESTful
? Select module do you want to use Add CRUD Operations
? Add Testing codes for Catching bugs early? Yes
? Add Vitest for Unit Testing? Yes
? Add Storybook for Visual Testing? Yes
? Add Playwright for End-To-End Testing? Yes
? Add ESLint for Code Linting? Yes
? Add Prettier for Code Formatting? Yes
```

--- a/packages/create-codes/templates/react/vitest/package.json
+++ b/packages/create-codes/templates/react/vitest/package.json
@@ -4,6 +4,7 @@
   },
   "devDependencies": {
     "@testing-library/react": "13.4.0",
+    "@testing-library/user-event": "14.4.3",
     "jsdom": "21.1.0",
     "node-fetch": "3.3.0",
     "vitest": "0.28.4"

